src/components/pages/StudentsPage/WelcomeMessage.css.tsx

```diff
@@ -0,0 +1,18 @@
+import { css } from '@emotion/react';
+
+const unpairedMessageContainer = css`
```

Comment:
We should be consistent with our naming. Let's use the same variable name as the filename. 
```suggestion
const welcomeMessageContainer = css`
```